<?xml version="1.0" encoding="UTF-8"?>
<!-- EN-Revision: 19661 -->
<!-- Reviewed: no -->
<sect1 id="zend.exception.using">
    <title>Usando as Exceções</title>

    <para>
        <classname>Zend_Exception</classname> é simplesmente a classe base para
        todas as exceções lançadas dentro do Zend Framework.
    </para>

    <example id="zend.exception.using.example">
        <title>Tratando uma Exceção</title>

        <para>
            O código a seguir demonstra como tratar uma exceção lançada em uma
            classe do Zend Framework:
        </para>

        <programlisting language="php"><![CDATA[
try {
    // Chamar Zend_Loader::loadClass() com uma classe inexistente irá causar
    // uma exceção para ser lançada em Zend_Loader:
    Zend_Loader::loadClass('nonexistantclass');
} catch (Zend_Exception $e) {
    echo "Tratando a exceção: " . get_class($e) . "\n";
    echo "Mensagem: " . $e->getMessage() . "\n";
    // Outro código para recuperar do erro
}
]]></programlisting>
    </example>

    <para>
        <classname>Zend_Exception</classname> pode ser usado como uma classe de exceção pega-tudo
        em um bloco de tratamento para capturar todas as exceções lançadas pelas classes do
        Zend Framework. Isso pode ser útil quando o programa não consegue se recuperar
        através do tratamento de um tipo específico de exceção.
    </para>

    <para>
        A documentação de cada componente e classe do Zend Framework irá conter informações
        específicas sobre os métodos para lançar exceções, as circunstâncias que fazem com
        que uma exceção seja lançada, e os diversos tipos de exceção que podem ser lançadas.
    </para>
</sect1>
<!--
vim:se ts=4 sw=4 et:
-->
